### PR TITLE
Calendar and styles

### PR DESCRIPTION
**Pull request by L4r1k**
I changed a lot of the styling around the app to allow for the navbar to be uniform across pages.
I also made the calendar dynamic and of course added a navbar

### CODE REVIEW

File: cleanslate/doma/static/css/base.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.navbar-light .navbar-nav .nav-link {
```
Comment (mboneil10) [inline cleanslate/doma/static/css/base.css]: Think the items between the navbar might need a bit more space.

File: cleanslate/doma/templates/base.html [hunk 2]
```diff
@@ -5,22 +5,53 @@
...
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="{% url 'finance' %}">Finance</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="{% url 'calendar' %}">Calendar</a>
```
Comment (mboneil10) [inline cleanslate/doma/templates/base.html:36]: Weird how the navbar moves on clicking the `calendar` tab.

**Review by mboneil10 — APPROVED** [review]

File: cleanslate/doma/templates/base.html (hunk 2 above)
Comment (L4r1k) [reply cleanslate/doma/templates/base.html:36]: Moves how? I don't see anything

File: cleanslate/doma/static/css/base.css (hunk 1 above)
Comment (L4r1k) [reply cleanslate/doma/static/css/base.css]: I'll give it a 15px padding either side

**Comment by mboneil10** [comment]
Yessss!

File: cleanslate/doma/templates/base.html (hunk 2 above)
Comment (mboneil10) [reply cleanslate/doma/templates/base.html:36]: You must have changed something. Or maybe I need to sleep more.

Comment (L4r1k) [reply cleanslate/doma/templates/base.html:36]: xD

**Review by onocy — APPROVED** [review]
Looks good!

**Pull request merged**